distil/primitives/load_edgelist.py

```diff
@@ -0,0 +1,260 @@
+                outputs = self._update_type_info(semantic_types, outputs, i)
+            elif 'https://metadata.datadrivendiscovery.org/types/SuggestedTarget' in semantic_types:
+                suggested_target_idx = i
+            elif 'https://metadata.datadrivendiscovery.org/types/PrimaryKey' in semantic_types:
+                outputs = self._update_type_info(semantic_types, outputs, i)
+        # fall back on suggested target
+        if target_idx == -1:
+            target_idx = suggested_target_idx
+
+        # flip the d3mIndex to be the df index as well
+        outputs = outputs.set_index('d3mIndex', drop=False)
+
+        remove_indices = set(range(num_cols))
+        remove_indices.remove(target_idx)
+        outputs = outputs.remove_columns(remove_indices)
+
+        logger.debug(f'\n{outputs.dtypes}')
+        logger.debug(f'\n{outputs}')
+
+        return base.CallResult(outputs)
+
+    @classmethod
+    def _update_metadata(cls, metadata: metadata_base.DataMetadata, resource_id: metadata_base.SelectorSegment) -> metadata_base.DataMetadata:
+        resource_metadata = dict(metadata.query((resource_id,)))
+
+        if 'structural_type' not in resource_metadata or not issubclass(resource_metadata['structural_type'], container.DataFrame):
+            raise TypeError("The Dataset resource is not a DataFrame, but \"{type}\".".format(
+                type=resource_metadata.get('structural_type', None),
+            ))
+
+        resource_metadata.update(
+            {
+                'schema': metadata_base.CONTAINER_SCHEMA_VERSION,
+            },
+        )
+
+        new_metadata = metadata_base.DataMetadata(resource_metadata)
+
+        new_metadata = metadata.copy_to(new_metadata, (resource_id,))
+
+        # Resource is not anymore an entry point.
+        new_metadata = new_metadata.remove_semantic_type((), 'https://metadata.datadrivendiscovery.org/types/DatasetEntryPoint')
+
+        return new_metadata
+
+    @classmethod
+    def _update_type_info(self, semantic_types: Sequence[str], outputs: container.DataFrame, i: int) -> container.DataFrame:
+        # update the structural / df type from the semantic type
+        if 'http://schema.org/Integer' in semantic_types:
+            outputs.metadata = outputs.metadata.update_column(i, {'structural_type': int})
+            outputs.iloc[:,i] = pd.to_numeric(outputs.iloc[:,i])
+        elif 'http://schema.org/Float' in semantic_types:
+            outputs.metadata = outputs.metadata.update_column(i, {'structural_type': float})
+            outputs.iloc[:,i] = pd.to_numeric(outputs.iloc[:,i])
+        elif 'http://schema.org/Boolean' in semantic_types:
+            outputs.metadata = outputs.metadata.update_column(i, {'structural_type': bool})
+            outputs.iloc[:,i] = outputs.iloc[:,i].astype('bool')
+
+        return outputs
+
+    @classmethod
+    def can_accept(cls, *, method_name: str, arguments: typing.Dict[str, typing.Union[metadata_base.Metadata, type]],
```

Comment:
`can_accept` is no longer a requirement on primitives so you can take it out.

Reply:
removed